workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/Utility/CommandLineUtilityTest.cs

```diff
@@ -6,6 +6,7 @@ namespace Improbable.Gdk.Core.EditmodeTests
     [TestFixture]
     public class CommandLineUtilityTest
     {
+        // TODO: Rewrite this.
```

Comment:
I assume this will happen in a separate pr?

Reply:
It'll be done before anything gets merged - that's why all the tests auto-fail right now